app/controllers/catalog_controller.rb

```diff
@@ -87,14 +87,14 @@ class CatalogController < ApplicationController
     # }
 
     config.add_facet_field 'domain_ssi', label: 'Domain'
-    config.add_facet_field 'author_ssim', label: 'Author'
-    config.add_facet_field 'genre_ssim', label: 'Genre'
-    config.add_facet_field 'subject_all_ssim', label: 'Subject'
-    config.add_facet_field 'publisher_ssim', label: 'Publisher'
-    config.add_facet_field 'year_available_itsi', label: 'Year Available', range: true
     config.add_facet_field 'community_root_name_ssi', label: 'Community'
     config.add_facet_field 'subcommunity_name_ssi', label: 'Subcommunity'
     config.add_facet_field 'collection_name_ssi', label: 'Collection'
+    config.add_facet_field 'author_ssim', label: 'Author', limit: true
```

Comment:
`limit:true` is what allows Blacklight to display the "more" value in the facet list that opens the pop-up window to view all facet values.